src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -86,6 +88,14 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_PERSON);
         }
 
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+
+        LocalDate dobToCheck = LocalDate.parse(editedPerson.getDob().value, formatter);
+        LocalDate adToCheck = LocalDate.parse(editedPerson.getAdmissionDate().value, formatter);
+        if (dobToCheck.isAfter(adToCheck)) {
+            throw new CommandException(Messages.MESSAGE_DOB_LATER_THAN_ADMISSION);
+        }
+
```

Comment:
Will it be better if dob > admission date only when dob or admission date is modified, instead of checking for every edit command?